src/components/atoms/secretInput.tsx

```diff
@@ -123,7 +124,13 @@ export const SecretInput = forwardRef<HTMLInputElement, SecretInputProps>((props
 
 	const inputType = isLocked ? "password" : "text";
 
-	const lockedIcon = isLocked ? UnlockedLockIcon : LockIcon;
+	const lockedIcon = (() => {
+		if (type === "password") {
+			return isLocked ? EyeCloseIcon : EyeIcon;
+		}
+
+		return isLocked ? UnlockedLockIcon : LockIcon;
+	})();
```

Comment:
Should be the opposite, open eye if we want to see the input text.

Also please use this icons:
https://fontawesome.com/icons/eye?f=classic&s=regular
https://fontawesome.com/icons/eye-slash?f=classic&s=regular

Reply:
changed